src/pspm_init.m

```diff
@@ -41,10 +41,27 @@
 % 2.1 check pspm version
 pspm_vers = pspm_version('check');
 % 2.2 check various settings
-global settings
-if ~isempty(settings) % initialise settings
-  settings = [];
+initial_paths = strsplit(path, pathsep);
+global settings;
+if isempty(settings)
+  settings = struct();
 end
+% Define the directories PsPM will need
+pspm_root = fileparts(which('pspm_init'));
+required_folders = {'backroom', 'ext', 'Import', 'pspm_cfg'};
```

Comment:
Perhaps it is useful to ask @dominikbach if `backroom` is expected here. I remember `backroom` is expected to be excluded in a previous commit but I am not fully certain.